CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 0.8.4 (Unreleased)
 
 IMPROVEMENTS:
+ * cli: Add node drain monitoring with new `-monitor` flag on node drain
```

Comment:
So this should be below the other since they are both under CLI but the length of this changelog entry is longer